### PR TITLE
Little FreeBSD build fix

### PR DESCRIPTION
**Pull request by devnexen**
Silenting few warnings and fix the unit test.

### CODE REVIEW

File: src/mi/backtrace.c [hunk 1]
```diff
@@ -25,7 +25,9 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.  */
 
 #ifndef UNW_REMOTE_ONLY
 
+#ifdef UNW_LOCAL_INLY
 #define UNW_LOCAL_ONLY
+#endif
```
Comment (am11) [inline src/mi/backtrace.c]: Maybe just:

```diff
- #ifndef UNW_REMOTE_ONLY
+ #if !defined(UNW_REMOTE_ONLY) && !defined(UNW_LOCAL_ONLY)

 #define UNW_LOCAL_ONLY
```

File: tests/forker.c [hunk 2]
```diff
@@ -24,7 +24,9 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.  */
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
+#if defined(__linux__)
 #include <alloca.h>
+#endif
```
Comment (am11) [inline tests/forker.c]: According to manpage, `alloca.h` is included by stdlib.h, so we do not need this on Linux either. I tried setting up FreeBSD CI and tried on both FreeBSD and Linux, it build without this header:

Linux: https://travis-ci.com/am11/libunwind/builds/133997124
FreeBSD: https://cirrus-ci.com/task/6552145913970688
Commit: https://github.com/am11/libunwind/commit/5bc1d67f7c2fcf4225f490fbcd68b27279840624

maybe we can just remove it? :)

**Comment by djwatson** [comment]
I can do one in the next month or so, I should have some free time shortly.

I'll merge everything that seems relevant (including this) to rc1.4, and promote to 1.4, and then cut an rc1.5.  

Thanks!

**Comment by djwatson** [comment]
Releases cut.

**Comment by am11** [comment]
Thanks a lot, @djwatson!

**Pull request merged**